src/wrappers/React.js

```diff
@@ -44,12 +44,13 @@ export default {
   methods: {
     mountReactComponent (component) {
       const Component = makeReactContainer(component)
+      const children = this.$slots.default ? { children: this.$slots.default } : {}
```

Comment:
We should probably check if `this.$slots.default !== undefined` (e.x. what if `this.$slots.default` is `0`?)